extensions/registration-service/src/main/java/org/eclipse/dataspaceconnector/registration/api/RegistrationApiController.java

```diff
@@ -25,7 +29,7 @@
 import jakarta.ws.rs.Produces;
 import jakarta.ws.rs.core.Context;
 import jakarta.ws.rs.core.HttpHeaders;
-import org.eclipse.dataspaceconnector.registration.authority.model.Participant;
+import org.eclipse.dataspaceconnector.registration.authority.dto.ParticipantDto;
```

Comment:
```suggestion
import org.eclipse.dataspaceconnector.registration.authority.model.ParticipantDto;
```
for alignment with EDC DTO package names

Reply:
okay, moved to model packages, if EDC prefer it this way. Thanks.